Coverage is not available on Solaris and those arguments are thus unknown to pytest.

Not suitable for upstream.

--- wheel-0.37.1/setup.cfg
+++ wheel-0.37.1/setup.cfg
@@ -53,7 +53,6 @@ distutils.commands =
 	bdist_wheel = wheel.bdist_wheel:bdist_wheel
 
 [tool:pytest]
-addopts = --cov --cov-config=setup.cfg
 testpaths = tests
 
 [coverage:run]
